narratives/ncov-19B_resurgence.md

```diff
@@ -0,0 +1,192 @@
+Here we see the SARS-CoV-2 global phylogeny colored by viral clade.
+Clades 19A & 19B, in purple and dark blue at the bottom of the tree, predominated early in the pandemic.
+However, these clades were soon eclipsed by viruses in clade 20.
+
+Clade 20 probably dominated for multiple reasons:
+This clade was exported globally, especially to Europe, as early as January 2020 where it spread rapidly undetected for over a month prior to lockdowns.
+Additionally, spike D614G is one of the clade-defining mutations, and this substitution likely confers [increased transmissibility](https://pubmed.ncbi.nlm.nih.gov/32697968/).
+
+Regardless, by June 2020, less than 10% of circulating virus were from clades 19A & 19B. By mid-September, frequency was no more than 1%.
+
+However, transmission did continue at low levels. Recently, we have observed a slight increase in the frequency of 19B.
+In December and January, we estimated 3% of all viruses to be 19B.
+
+This resurgence of a viral clade that had largely died out, especially during the worst phase of the pandemic, is surprising.
+
+To explore this phenomenon, we built a phylogeny enriched for recent 19A & 19B viruses.
+In this report, we use that phylogeny to explore the genetic composition of currently circulating 19B virus.
+
+<!-- There is NO right-side text on this slide -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- SLIDE 2 -->
+# [Recent 19B are present across the globe](https://nextstrain.org/groups/blab/ncov/19B?c=country&d=tree,map&dmin=2020-11-01&label=clade:19B&p=grid&legend=open)
+
+<!-- This is the left-side text -->
+Here we show all available sequences in clade 19B sampled from November 2020 onward.
+These sequences were collected from across the globe in North America, South America, Europe, Africa, and the Middle East.
+This suggests that transmission of this clade is ongoing in multiple locations.
+
+Note: Our analysis is biased by the available data.
+For example, almost half of all recent 19B sequences come from the UK, which is sequencing far more SARS-CoV-2 than any other country.
+
+<!-- There is NO right-side text on this slide -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- SLIDE 3 -->
+# [Independent emergences of Spike 614G & 614N](https://nextstrain.org/groups/blab/ncov/19B?c=gt-S_614&d=tree&p=full&legend=open)
+
+<!-- This is the left-side text -->
+Coloring the tree by amino acid at spike 614, we observe, multiple independent emergences of 614G and 614N in 19B & 19A.
+_In vitro_ ([Zhang et al](https://www.nature.com/articles/s41467-020-19808-4), [Yurkovetskiy et al](https://www.sciencedirect.com/science/article/pii/S0092867420312290), [Korber et al](https://pubmed.ncbi.nlm.nih.gov/32697968/)) and _in vivo_ work in mice and hamsters ([Hou et al](https://science.sciencemag.org/content/370/6523/1464), [Plante et al](https://www.nature.com/articles/s41586-020-2895-3)) demonstrate increased infectivity with the 614G substitution.
```

Comment:
Do we know anything about 614N? If not, we should maybe just add "No analysis exploring of the impact of 614N is currently available." (or similar)